build.gradle

```diff
@@ -10,28 +11,12 @@ spotless {
                 'diy_asylum_frontend/*.gradle',
                 'diy_asylum_frontend/*.md',
                 'formfiller/*.gradle',
-                'formfiller/*.md'
+                'formfiller/*.md',
+                'formserver/*.gradle',
+                'formserver/*.md'
 
         trimTrailingWhitespace()
         indentWithSpaces()
         endWithNewline()
     }
-    java {
-        googleJavaFormat()
-    }
-    format 'styling', {
-        target 'diy_asylum_frontend/src/**/*.css',
-                'diy_asylum_frontend/src/**/*.scss'
-        prettier().config(['parser': 'css'])
-    }
-    format 'javascript', {
```

Comment:
Put this back

Reply:
It's in the sub projects for now. I was struggling to get this working quite right. I need to read up a bit more before and see if I can bring it back into root rather than define it per sub

Reply:
https://diyasylum.atlassian.net/browse/DIYAS-24 will address this